Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md

```diff
@@ -0,0 +1,102 @@
+This playbook investigates and remediates a potential phishing incident. It engages with the user that triggered the incident while investigating the incident itself.
+Note: Final remediation tasks are always decided by a human analyst. 
```

Comment:
Missing a reference to that this is for 6.1 and that we are using indicator extraction rules

Reply:
Done. @julieschwartz18 can you please take a look here?
https://github.com/demisto/content/pull/16327/commits/b0328c57e0839615a6fbfd15ac2e817f6cc821ae